RELEASE/scripts/postsool.ash

```diff
@@ -743,11 +743,17 @@ void handlePostAdventure()
 		if(doML)
 		{
 			// Catch when we leave Smut Orc, allow for being "side tracked" buy delay burning
-			if((0 < have_effect($effect[Driving Intimidatingly])) && (debuffAsdonDelay > 1))
+			if((have_effect($effect[Driving Intimidatingly]).to_int() > 0) && (debuffAsdonDelay >= 2))
```

Comment:
Why `.to_int()` something that's already an int?

Reply:
Ah, yeah that shouldn't still be in there. It is from when I was trying to figure out things.